client/state/notices/middleware.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { truncate } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { successNotice, errorNotice } from 'state/notices/actions';
+import { getSitePost } from 'state/posts/selectors';
+import {
+	POST_DELETE_FAILURE,
+	POST_DELETE_SUCCESS,
+	POST_SAVE_SUCCESS
+} from 'state/action-types';
+
+export const OBSERVERS = {
```

Comment:
> documented style

It feels like that section could use some updating. This feels qualitatively different to me than something like is in the example:

```
const MAX_ITEMS = 23;
```

It feels incidentally constant as a side-effect of how we have chosen to organize the code. In fact, if we had used a switch structure instead this would be silly here. On the other hand, a literal constant is basically always a constant no matter where we use it.

Do we even use this outside of this file anywhere besides in the tests?